wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

```diff
@@ -26,10 +26,14 @@ If discussion is no longer progressing in the direction of a compromise, or is o
 
 If the vetoer is no longer a Beatmap Nominator and no mediation has been requested, the veto will be dismissed.
 
-During mediation, the Beatmap Nominators and members of the NAT will vote for whether to uphold or dismiss the veto anonymously. Once initiated, this process can take anywhere from a day up to a week depending on how quickly participants make their decisions.
+### Veto Mediation
```

Comment:
idk why this is a subheading. could rename the enclosing section to "veto mediation"